src/main/java/uk/gov/ons/ssdc/caseprocessor/service/UacService.java

```diff
@@ -78,11 +79,17 @@ public boolean existsByQid(String qid) {
   }
 
   public void createLinkAndEmitNewUacQid(
-      Case caze, String uac, String qid, UUID correlationId, String originatingUser) {
+      Case caze,
+      String uac,
+      String qid,
+      Map<String, String> uacMetadata,
```

Comment:
Should be `Object` or `String` to allow future flexibility beyond flat key:value pairs of strings